packages/pagination/src/styled/pagination/StyledPage.ts

```diff
@@ -91,7 +94,7 @@ export const StyledPage = styled.li.attrs({
   visibility: ${props => props.hidden && 'hidden'};
   margin-left: ${props => `${props.theme.space.base}px`};
   border-radius: ${props => props.theme.borderRadii.md};
-  cursor: pointer;
+  cursor: ${props => !(props as any).disabled && 'pointer'};
```

Comment:
These props are typed `as any` because `StyledCursor` (button) composes from `StyledPage` (not button):

```jsx
const StyledCursor = styled(StyledPage)
```

An alternative solution is to use Garden’s buttons from the `react-buttons` package. I went with what was discussed in person, but I am open to suggestions.